### ISSUE TITLE
VersionConflict error messages should include the conflicting package's name

### ISSUE THREAD

**Issue by underyx** [coala-analyzer/coala#2683]

**Comment by AbdealiJK** [comment]
Does it not ?
I honestly don't want to mess up my systemt o reproduce, but @Asnelchristian reported on the gitter channel yesterday that he got:


[ERROR][20:34:09] There is a conflict in the version of a dependency you have installed and the requirements of coala. This may be resolved by creating a separate virtual environment for coala or running `pip install nltk==3.1.*`. Be aware that the latter solution might break other python packages that depend on the currently installed version.

Which does say that nltk was the issue. Or is this about adding nltk to the first line like:

[ERROR][20:34:09] There is a conflict in the version of a dependency you have installed and the requirements of coala (package: nltk). This may be resolved by creating a separate virtual environment for coala or running `pip install nltk==3.1.*`. Be aware that the latter solution might break other python packages that depend on the currently installed version.

**Comment by sils1297** [comment]
It's not obvious which of our packages lead to nltk as a dependency and I
think that package should be shown.

**Comment by AbdealiJK** [comment]
Ah, so you mean the bear name should be shown ?
Or by which of our packages do you mean coala-util or coala-bears or coala ?

Doesn't our LogPrinter show this ? The builtin logging module does show which application created the log iirc

**Comment by sils1297** [comment]
yep bear name, also this shouldn't block coala from running completely right? Just that bear.

**Comment by ankitjavalkar** [comment]
Hi, can someone help me get started with this?
I see that the VersionConflict Message is located in constants.py but does the exception already know which Bear caused it? How do I obtain the Bear name to print in the message.
Apologies if the question sounds naive, this is my first difficulty/low ticket (only dealt with newcomer tickets before)
CC @AbdealiJK @sils1297

**Comment by Nosferatul** [comment]
In coalib/bears/Bear.py you have a property `name` that return the name of the used bear. Maybe you can pass the bear as an argument for `get_exitcode` function from Exceptions.py ? I hope this will help you!

**Comment by sils1297** [comment]
Maybe the exception has a member that holds that info?

**Comment by Nosferatul** [comment]
@ankitjavalkar are you working on it?

**Comment by Nosferatul** [comment]
unassigned due inactivity

**Comment by Projjol** [comment]
Hi, could I take a shot at this?

**Comment by jackthepirate6** [comment]
@Projjol Are you working on it ? If not pls let me take the issue

**Comment by Makman2** [comment]
Unassigning due to inactivity.

**Comment by Nosferatul** [comment]
unassigned due to inactivity

**Comment by gabi0348** [comment]
Hi, could I take a shot at this?
